Mathlib/Analysis/CStarAlgebra/Spectrum.lean

```diff
@@ -240,7 +235,6 @@ lemma nnnorm_apply_le (φ : F) (a : A) : ‖φ a‖₊ ≤ ‖a‖₊ := by
       exact this <| .star_mul_self x
     intro s hs
     suffices this : spectralRadius ℂ (ψ s) ≤ spectralRadius ℂ s by
-      -- changing the order of `rw`s below runs into https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/weird.20type.20class.20synthesis.20error/near/421224482
```

Comment:
I suspect, but I'm not positive, that this synth pending issue will no longer plague stuff involving `Unitization`. I checked that this comment is no longer necessary.